### PR TITLE
#1386: Bug fixes

### PR DESCRIPTION
**Pull request by oderayi**
- Fixed API spec and handler for bulk transfers POST endpoint

### CODE REVIEW

File: src/package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mojaloop-simulator",
-  "version": "10.6.1",
+  "version": "10.6.2",
```
Comment (akfranco) [inline src/package-lock.json]:
```suggestion
  "version": "10.6.4",
```

File: src/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mojaloop-simulator",
-  "version": "10.6.1",
+  "version": "10.6.2",
```
Comment (akfranco) [inline src/package.json]:
```suggestion
  "version": "10.6.4",
```

**Review by akfranco — COMMENTED** [review]
Hey @oderayi we've been doing a bunch of PR's and commits to this repo lately, sorry for getting in your way here. I've bumped up your versions to where we're at now.

**Comment by oderayi** [comment]
> Hey @oderayi we've been doing a bunch of PR's and commits to this repo lately, sorry for getting in your way here. I've bumped up your versions to where we're at now.

Thanks for the update @KamuelaFranco. I'm just going to hold on a bit on this one until the coast is clear.

**Pull request closed without merging**